.goreleaser.yaml

```diff
@@ -0,0 +1,303 @@
+      - '--label=org.opencontainers.image.licenses=Apache-2.0'
+      - '--label=org.opencontainers.image.name=jvs-public-key'
+      - '--label=org.opencontainers.image.revision={{ .FullCommit }}'
+      - '--label=org.opencontainers.image.source={{ .GitURL }}'
+      - '--label=org.opencontainers.image.title=jvs-public-key'
+      - '--label=org.opencontainers.image.version={{ .Version }}'
+  -
+    ids:
+    - public-key
+    use: 'buildx'
+    goos: 'linux'
+    goarch: 'arm64'
+    image_templates:
+      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-public-key:{{ .Version }}-arm64'
+      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-public-key:latest-arm64'
+    build_flag_templates:
+      - '--build-arg=APP=public-key'
+      - '--platform=linux/arm64'
+      - '--pull'
+      - '--label=org.opencontainers.image.created={{ .CommitTimestamp }}'
+      - '--label=org.opencontainers.image.description=Why do you care?'
+      - '--label=org.opencontainers.image.licenses=Apache-2.0'
+      - '--label=org.opencontainers.image.name=jvs-public-key'
+      - '--label=org.opencontainers.image.revision={{ .FullCommit }}'
+      - '--label=org.opencontainers.image.source={{ .GitURL }}'
+      - '--label=org.opencontainers.image.title=jvs-public-key'
+      - '--label=org.opencontainers.image.version={{ .Version }}'
+
+
+docker_manifests:
+  -
+    name_template: '{{ .Env.CONTAINER_REGISTRY }}/jvs-cert-rotation:{{ .Version }}'
+    image_templates:
+      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-cert-rotation:{{ .Version }}-amd64'
+      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-cert-rotation:{{ .Version }}-arm64'
+  -
+    name_template: '{{ .Env.CONTAINER_REGISTRY }}/jvs-cert-rotation:latest'
+    image_templates:
+      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-cert-rotation:latest-amd64'
+      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-cert-rotation:latest-arm64'
+  -
+    name_template: '{{ .Env.CONTAINER_REGISTRY }}/jvs-justification:{{ .Version }}'
+    image_templates:
+      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-justification:{{ .Version }}-amd64'
+      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-justification:{{ .Version }}-arm64'
+  -
+    name_template: '{{ .Env.CONTAINER_REGISTRY }}/jvs-justification:latest'
+    image_templates:
+      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-justification:latest-amd64'
+      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-justification:latest-arm64'
+  -
+    name_template: '{{ .Env.CONTAINER_REGISTRY }}/jvs-public-key:{{ .Version }}'
+    image_templates:
+      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-public-key:{{ .Version }}-amd64'
+      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-public-key:{{ .Version }}-arm64'
+  -
+    name_template: '{{ .Env.CONTAINER_REGISTRY }}/jvs-public-key:latest'
+    image_templates:
+      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-public-key:latest-amd64'
+      - '{{ .Env.CONTAINER_REGISTRY }}/jvs-public-key:latest-arm64'
+
+
+archives:
+  - format: 'tar.gz'
+    name_template: 'abcxyz_jvs_{{ .Version }}_{{ .Os }}_{{ .Arch }}'
```

Comment:
What's the goal with prefixing with abcxyz?

Reply:
Thought it's clearer... Removed now.